src/containers/Tables/BidTable/BidTableRow.tsx

```diff
@@ -1,18 +1,20 @@
 //- Styles Imports
 import styles from './BidTable.module.scss';
-import moreIcon from 'assets/more-vertical.svg';
+import moreIcon from 'assets/more-horizontal.svg';
 import { DollarSign, X } from 'react-feather';
 
 //- Components Imports
-import { Artwork, OptionDropdown } from 'components';
+import { Artwork, FutureButton, OptionDropdown, Tooltip } from 'components';
 import { Option } from 'components/Dropdowns/OptionDropdown/OptionDropdown';
 
 //- Library Imports
 import { ethers } from 'ethers';
 import { Domain } from 'lib/types';
 
 //- Constants Imports
-import { TOKEN } from './BidTableRow.constants';
+import { getNetworkZNA } from 'lib/utils';
+import { useHistory } from 'react-router-dom';
+import { ROUTES } from 'constants/routes';
```

Comment:
nit: can we rearrange these imports a little - `getNetworkZNA` under `Library imports`, etc.

Reply:
Good call, have addressed in latest commit!